website/docs/upgrades/version_base.md

```diff
@@ -0,0 +1,19 @@
+---
+id: version_base
+title: version_base description
+---
```

Comment:
It would be good to add `'upgrades/version_base'` and `'upgrades/1.1_to_1.2/hydra_main_config_path'` to the `website/sidebars.js` file so that these new markdown files can be found via the hydra docs sidebar.